app/editor/src/features/content/list-view/interfaces/IContentListFilter.ts

```diff
@@ -24,4 +24,5 @@ export interface IContentListFilter {
   topStory: boolean;
   homepage: boolean;
   sort: ISortBy[];
+  contentType?: string;
```

Comment:
Don't add a single `contentType` filter.  We already support passing an array of `contentTypes`

Reply:
Good catch, just clean it up, thanks